targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCpp/Private/Tests/PlayFabCppTests.cpp.ejs

```diff
@@ -1,9 +1,8 @@
 <%- copyright %>
 
-#include "CppTests/PlayFabCppTests.h"
+#include "Tests/PlayFabCppTests.h"
```

Comment:
In the case of the CPP interface tests, it's important that the tests exist OUTSIDE of the cpp interface module, because that's an important part of testing them. The CPP module MUST work when accessed from the outside, and the tests have to demonstrate that.

The CPP tests have been confirmed as working already in that state, so let's not move these tests into the CPP module.